### DISCUSSION TITLE
Randomly distribution in pseudo_gravity_packing

### DISCUSSION THREAD

**Discussion by Jinhe-Zhang** [PMEAL/porespy#822] [Ideas]
The diameter of the spheres is a constant number in the current generated models. I was wondering if you plan to continually develop this model, in which spheres can be different sizes (and not overlapping)?

**Comment by jgostick** [comment]
Hi @Jinhe-Zhang, 
No plans at the moment. Originally I toyed with the idea of allowing random sizes, but it turned out that it was insanely computationally intensive (because of the naive way the spheres are added).

**Reply by Jinhe-Zhang** [reply]
Okay, thanks!
